docs/guides/Smart Contracts/4.deploy.md

```diff
@@ -49,41 +43,13 @@ Currently we recommend the Jungle Testnet as the Ultra is still building infrast
         <span><tt>ultra.eosio test YOUR_PUBLIC_KEY --transfer --gift-ram-kbytes 1024000 -p ultra.eosio</tt></span>
         <span>4. Deploy your contract using instructions below</span>
     </template>
-    <template #Jungle-Testnet>
-        <b>Jungle Testnet</b>
-        <span>Visit <a href="https://monitor4.jungletestnet.io/">https://monitor4.jungletestnet.io/</a> to get started.</span>
-        <em>This is a multi-step process before publishing.</em>
-        <hr />
-        <b>Create an Account</b>
-        <span>1. Click Create Account</span>
-        <span>2. Fill out account name with a 12 character name</span>
-        <span>3. Paste a Public Key in Owner & Active Keys</span>
-        <span>4. Click Create</span>
-        <hr />
-        <b>Get Tokens</b>
-        <span>1. Click Faucet</span>
-        <span>2. Fill out account name</span>
-        <span>3. Click Send Coins</span>
-        <hr />
-        <b>Get RAM Resource</b>
-        <span>1. Open VSCode and Command Palette (F1) and type <em>Ultra: Create Transaction</em></span>
-        <span>2. Select jungle testnet</span>
-        <span>3. Enter Wallet Password</span>
-        <span>4. Lookup eosio</span>
-        <span>5. Select action <em>buyrambytes</em></span>
-        <span>6. Look at your .wasm file properties to determine the RAM you need. Buy extra for storing table data. You will want to lookup the total amount of bytes your .wasm file has.</span>
-        <span>7. Fill out the form, and buy some RAM. payer and receiver should be the same.</span>
-        <span>8. Ensure that the transaction is successful</span>
-        <hr />
-        <em>Follow the rest of the instructions below for publishing.</em>
-    </template>
     <template #Ultra-Testnet>
         <b>Ultra Testnet</b>
-        <em>The Ultra Testnet does not currently allow publishing contracts at this time.</em>
-        <em>However, it will soon be available. Q3 2023.</em>
+        <span>1. Use ultra <a href="https://faucet.testnet.app.ultra.io/">faucet</a> to create a non-eba account and receive tokens</span>
+        <span>2. Use ultra <a href="https://toolkit.ultra.io/">toolkit</a> to buy ram for your account</span>
     </template>
     <template #Ultra-Mainnet>
-        <b>Contact <a href="mailto:developers@ultra.io">developers@ultra.io</a></b>
+        <span>Use ultra <a href="https://toolkit.ultra.io/">toolkit</a> to create a non-eba account and buy ram</span>
```

Comment:
Here you will also need to mention you need an EBA account first, add a step to create account with https://ultra.io/ client

Reply:
And for the toolkit clarify that you need to log in through Ultra Wallet Extension only and use their EBA account credentials